### PR TITLE
Adding support for android intents

### PR DESCRIPTION
**Pull request by nate-moo**

### CODE REVIEW

**Comment by AbdullahM0hamed** [comment]
`anime dl naruto --play android -c 1 -p animefreak` despite being 1 episode long (movie), still waits for input

**Comment by nate-moo** [comment]
Unless we do some checks to see how many episodes are in the show that may be a little difficult to fix as the current logic only looks for if an -e argument is applied and what it's contents are

File: anime_downloader/util.py [hunk 1]
```diff
@@ -222,13 +222,19 @@ def print_episodeurl(episode):
...
             f'--title={title}', 
             f'--referrer={episode.source().referer}', 
             f'--user-agent={get_random_header()["user-agent"]}', 
             episode.source().stream_url])
+    elif player == "android":
+        if episodes == None or ':' in episodes:
+            p = subprocess.Popen(['am', 'start', '-a', 'android.intent.action.VIEW', '-t', 'video/*', '-d', f'{episode.source().stream_url}'])
+            input("Press enter to continue\n")
+        else:
+            p = subprocess.Popen(['am', 'start', '-a', 'android.intent.action.VIEW', '-t', 'video/*', '-d', f'{episode.source().stream_url}'])
```
Comment (AbdullahM0hamed) [inline anime_downloader/util.py:237]: Don't repeat this line - do it before the if statement, and only put the input in the if statement

File: anime_downloader/commands/dl.py [hunk 2]
```diff
@@ -130,7 +130,7 @@ def command(ctx, anime_url, episode_range, url, player, skip_download, quality,
             util.print_episodeurl(episode)
 
         if player:
-            util.play_episode(episode, player=player, title=f'{anime.title} - Episode {episode.ep_no}')
```
Comment (AbdullahM0hamed) [inline anime_downloader/commands/dl.py:133]: Add something like `episode_ramge = f"0:{len(animes)}" if not episode_ramge else episode_range` before this line

**Review by AbdullahM0hamed — CHANGES_REQUESTED** [review]

File: anime_downloader/commands/dl.py (hunk 2 above)
Comment (nate-moo) [reply anime_downloader/commands/dl.py:133]: Had to modify util.py to get this to fully work

File: anime_downloader/util.py [hunk 3]
```diff
@@ -221,20 +221,24 @@ def print_episodeurl(episode):
...
+            f'--title={title}',
+            f'--referrer={episode.source().referer}',
+            f'--user-agent={get_random_header()["user-agent"]}',
             episode.source().stream_url])
+    elif player == "android":
+        p = subprocess.Popen(['am', 'start', '-a', 'android.intent.action.VIEW', '-t', 'video/*', '-d', f'{episode.source().stream_url}'])
+        if episodes == None or ':' in episodes:
+            if "0:1" in episodes:
+                pass
+            else:
```
Comment (AbdullahM0hamed) [inline anime_downloader/util.py:236]: Unnecessary if-else, negate the condition by doing `if "0:1" not in episodes` instead

**Review by AbdullahM0hamed — CHANGES_REQUESTED** [review]

File: anime_downloader/util.py (hunk 3 above)
Comment (nate-moo) [reply anime_downloader/util.py:236]: Didn't think about that, lemme go fix

**Comment by AbdullahM0hamed** [comment]
Works, merging

**Pull request merged**